suzieq/poller/worker/services/routes.py

```diff
@@ -48,6 +48,8 @@ def _clean_eos_data(self, processed_data, _):
                 entry['oifs'] = len(entry['nexthopIps']) * \
                     ['_nexthopVrf:default']
             entry['protocol'] = entry['protocol'].lower()
+            if entry['protocol'] == 'connected':
```

Comment:
In the commit you are saying that you should do this change only when 0.0.0.0 as nexthopIps. Here you are always updating it (hence you are not checking if `nexthopIps == '0.0.0.0'`. Is it the right thing to do?

Reply:
the point is that the nexthopIps list should be always empty if the protocol is connected. That was just an explanation of the bug, there are some cases where we put `0.0.0.0` as nexthopIps even if the protocol is connected, so here we just set nexthopIps as empty every time we have `protocol=connected` , it doesn't metter  checking if nexthopIps is equal to `0.0.0.0` or not.